### PR TITLE
Support to Multiple Languages 

### PR DESCRIPTION
**Pull request by miharbi**
Hi, yes I know is a lote to check, but I think is something that will make the CMS more attractive to use. 

There was a small bug too in event creation and edit.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Can you revert all changes to the file permissions please?

**Comment by miharbi** [comment]
Im here, sorry, did I do any?

**Comment by miharbi** [comment]
ok, I think now I got it

**Comment by GrahamCampbell** [comment]
All the file permissions are still wrong.

**Comment by GrahamCampbell** [comment]
Also, I can't merge this without translations on the Credentials component too.

**Comment by miharbi** [comment]
Ok, Yes You are right, let me check all the components. I'm just doing a proposal. I don't understand that about the permissions, could you say me what I need to do?

**Comment by GrahamCampbell** [comment]
You've changed all the file permissions to 755. You must not change them please.

**Comment by GrahamCampbell** [comment]
Example `app/Http/Controllers/CommentController.php 100644 → 100755`. That can't happen.

**Comment by miharbi** [comment]
Ok, I did not notice that, no problem, I'll fix that in a while, I don't know why is this happening, let me check

**Comment by GrahamCampbell** [comment]
Probably because you chmoded everything to get it to work on our websever? Instead, just chown it to the correct user.

**Comment by miharbi** [comment]
hey did you check the commits?

**Comment by GrahamCampbell** [comment]
Could you please send all non-language related changes in a separate PR?

**Comment by miharbi** [comment]
I think that all of those aren't related with another stuff, look the  Sep 7, 2015 commit

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -110,7 +110,7 @@ public function store($postId)
         ]);
 
         if (CommentRepository::validate($input, array_keys($input))->fails()) {
-            throw new BadRequestHttpException('Your comment was empty.');
```
Comment (GrahamCampbell) [inline app/Http/Controllers/CommentController.php]: Exceptions must always be in english.

Comment (miharbi) [inline app/Http/Controllers/CommentController.php]: Well man, sorry, I was very motivated those days and with a lot of free time, but, I'm kinda busy right now. I'm sure that you had been busy and that is why you take all this time to answer. I'll understand if you wanna dismiss my pull requests.

**Comment by epileftro85** [comment]
hi, how can i help with this? i can make the spanish translation if you want to.

**Comment by miharbi** [comment]
I made all the translations, but he wanted the Exceptions in english, can you check the Exceptions, and change those to english or revert my changes on it?

**Comment by epileftro85** [comment]
i think it would be nice if you can have two translation in exeptions too, well i am just saying

**Comment by miharbi** [comment]
He(@GrahamCampbell) have right on this point, no matter what, english is the default language, it is a try of Multiple Languages, the idea is use more than one language in the frontend, ok, but by default all the Exceptions have to be in english. It was my fail, I hope to have time in the next week for work on it, but if you can solve that, it will be great!

**Comment by GrahamCampbell** [comment]
Thank you, but I don't have the time or energy to support this and keep it up to date.

**Pull request closed without merging**